public/views/partials/lib/helpers/can_do_or_unauthorized.liquid

```diff
@@ -0,0 +1,19 @@
+{% comment %}
+  Renders 401 if the the requester does not have right to do something on the given entity.
+
+  Params:
+    - requester Object
+      The requester must has a string array field called 'permissions'
+    - entity Object
+    - do String
+    - access_callback String
+      The callback function that checks the request. By default searches that the `requester`
+      has `do` permission or not.
+{% endcomment %}
+{% liquid
+  function can = 'modules/permission/lib/helpers/can_do', requester: requester, entity: entity, do: do, access_callback: access_callback
+  unless can
+    render 'modules/permission/theme/401'
```

Comment:
this is fine for now, but in the near future we should support template overrides here.
creating a ticket for the next sprint 👍

Reply:
sure and we need to discuss where to put these kind of codes